airflow_dag_samples/embedded_dataset_dag/sdk_calls.py

```diff
@@ -24,10 +25,10 @@ def read_by_line(dataset_filepath):
             raise Exception("Failed to save dataset: ",
                             dataset_response.exception)
 
-        return dataset_response.datasetId
+        return dataset_response.dataset_id
```

Comment:
Question: Am I off here, I'm seeing that `dataset_id` doesn't exist in class `SaveDatasetResponse`
<img width="521" alt="Screen Shot 2023-10-26 at 9 19 23 AM" src="https://github.com/graphgrid/graphgrid-sdk-python-examples/assets/6307559/de4155ee-7820-41be-b2c3-b9f8f8d0f122">

Do we need to update:
```
class SaveDatasetResponse(SdkServiceResponse):
    """Define class representing a save dataset api call response."""
    bucket: str
    key: str
    datasetId: str # this ?
````

Reply:
It has been updated from datasetId to dataset_id in the code, I think the sdk package just needs to be built and uploaded with those changes since that's what this project is pulling from

Reply:
ah that makes sense nvm me then